docker/sorry-cypress/nginx.conf

```diff
@@ -0,0 +1,125 @@
+http {
+    server {
+        listen 80;
+        server_name sorry-cypress.ase.cit.tum.de;
+
+        location / {
+            return 301 https://$host$request_uri;
+        }
+    }
+
+    server {
+        listen 443 ssl;
+        server_name sorry-cypress.ase.cit.tum.de;
```

Comment:
If possible configuring these via env vars would also be great. Would make possible future URL changes easier ;)

Reply:
As far as I can see, there is no easy/good way to use env vars in nginx files. You can create templates, that seem to support this, but they don't support other things. Maybe I'm missing something here